packages/components/tests/integration/components/hds/alert/index-test.js

```diff
@@ -15,11 +15,7 @@ module('Integration | Component | hds/alert/index', function (hooks) {
     resetOnerror();
   });
 
-  test('it renders the alert container', async function (assert) {
-    await render(hbs`<Hds::Alert @type="inline" />`);
-    assert.dom(this.element).exists();
-  });
-  test('it should render with a CSS class that matches the component name', async function (assert) {
+  test('it renders the component with a CSS class that matches the component name', async function (assert) {
```

Comment:
probably we should stick to one of the two:
1. `it renders blah blah`
2. `it should render blah blah`

(no strong opinions on one or the other, but I see that for most the other tests we tend to use `should`).

Reply:
We have 63 instances of "it renders the component with.."
We have 11 instances of "it should render the component with..." 

I'll update to use the majority since the point was consistency, good spot!